app/scripts/modules/core/src/instance/Instance.tsx

```diff
@@ -0,0 +1,44 @@
+import * as React from 'react';
+import autoBindMethods from 'class-autobind-decorator';
+
+import { IInstance } from 'core/domain';
+
+export interface IInstanceProps {
+  instance: IInstance;
+  active: boolean;
+  highlight: string;
+  onInstanceClicked(instance: IInstance): void;
+}
+
+@autoBindMethods
+export class Instance extends React.Component<IInstanceProps> {
+  public render() {
```

Comment:
Not sure we have ever discussed ordering of methods, but I like the pattern of render at the bottom.